### ISSUE TITLE
Max length of help output?

### ISSUE THREAD

**Issue by endquote** [tj/commander.js#540]
I have a tool with a fair number of arguments and subcommands, and the help is getting cut off.

![screenshot 2016-05-18 11 19 11](https://cloud.githubusercontent.com/assets/330010/15370150/662ceb60-1cea-11e6-88fa-6ac134755bc2.png)

https://github.com/endquote/laundry

**Comment by zhiyelee** [comment]
in node v6? https://github.com/tj/commander.js/issues/530

**Comment by endquote** [comment]
6.1.0...

**Comment by derek-ellison** [comment]
A good work around for this, although time consuming, is to build your own help menu using the program.on('--help') functionality. It's time consuming as I said, but you do have a lot more flexibility and you avoid this bug with the newer versions of Node.

```
program.on('--help', function() {
      console.log('--stuff-here       things');
});
```

**Comment by creeperyang** [comment]
Cut off +1, `node v6.2` and `command v2.9.0`.

<img width="601" alt="2016-06-14 4 04 12" src="https://cloud.githubusercontent.com/assets/8046480/16035432/af5a056a-3249-11e6-9a75-0114934252be.png">

**Comment by GCheung55** [comment]
Cut off here as well. `node v6.1.0` and `commander v2.9.0`.

**Comment by tyge68** [comment]
I have the same issue with node 6.2.0 and commander 2.9.0

**Comment by tyge68** [comment]
It appear that the cut off is due to the program.exit(0) which doesn't give time for the stdout.write operation to complete. Similar issue have been reported. So most likely the implementation should avoid to force exit, instead it should end in a normal flow without this call.

**Comment by shadowspawn** [comment]
The comments in #540 suggest this was fixed in node 6.6.0.

This issue has not had any activity in over six months. It isn't likely to get acted on due to this report.

Feel free to open a new issue if it comes up again, with new information and renewed interest.
 
Thank you  for your contributions.

**Comment by shadowspawn** [comment]
Future readers: this may still be an issue on Windows TTY which are asynchronous according to:
- https://nodejs.org/api/process.html#process_a_note_on_process_i_o

Please open a new issue if you are seeing truncated help.
